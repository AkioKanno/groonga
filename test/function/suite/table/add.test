register table/table
table_create Shops TABLE_PAT_KEY ShortText
column_create Shops name COLUMN_SCALAR ShortText
column_create Shops tags COLUMN_VECTOR ShortText

table_create Events TABLE_NO_KEY
column_create Events value COLUMN_SCALAR ShortText

add Shops '{"_key":"kuriko-an","tags":["たいやき","養殖"]}'
select Shops

add Events '{"value":"田無店新規開店!"}'
select Events
