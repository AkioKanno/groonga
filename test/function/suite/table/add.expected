register table/table
[[0,0.0,0.0],true]
table_create Shops TABLE_PAT_KEY ShortText
[[0,0.0,0.0],true]
column_create Shops name COLUMN_SCALAR ShortText
[[0,0.0,0.0],true]
column_create Shops tags COLUMN_VECTOR ShortText
[[0,0.0,0.0],true]
table_create Events TABLE_NO_KEY
[[0,0.0,0.0],true]
column_create Events value COLUMN_SCALAR ShortText
[[0,0.0,0.0],true]
add Shops '{"_key":"kuriko-an","tags":["たいやき","養殖"]}'
[[0,0.0,0.0],true]
select Shops
[
  [
    0,
    0.0,
    0.0
  ],
  [
    [
      [
        1
      ],
      [
        [
          "_id",
          "UInt32"
        ],
        [
          "_key",
          "ShortText"
        ],
        [
          "name",
          "ShortText"
        ],
        [
          "tags",
          "ShortText"
        ]
      ],
      [
        1,
        "kuriko-an",
        "",
        [
          "たいやき",
          "養殖"
        ]
      ]
    ]
  ]
]
add Events '{"value":"田無店新規開店!"}'
[[0,0.0,0.0],true]
select Events
[
  [
    0,
    0.0,
    0.0
  ],
  [
    [
      [
        1
      ],
      [
        [
          "_id",
          "UInt32"
        ],
        [
          "value",
          "ShortText"
        ]
      ],
      [
        1,
        "田無店新規開店!"
      ]
    ]
  ]
]
